applications/visualizer/frontend/package.json

```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```

Comment:
question (non-blocking): Previously we had a limit of 0 warnings on the linting which was used to understand if the linting step was passing or not. How does that work with biome now? 
For context, running `npm run lint` in the current branch gives me:

```
Checked 56 files in 18ms. No fixes applied.
Found 38 errors.
```

Reply:
The linter wasn't used by any github action before (and is still not in this version). The first source of truth for the validity of the code right now is the fact that it compiles. Biome here is considering warning as non blocking, which is fine at the moment. Later, we can add the `--error-on-warning` argument (https://biomejs.dev/linter/) if we need to.